docs/superpowers/specs/2026-04-24-recommendation-dedup-design.md

```
@@ -0,0 +1,74 @@
+```
+
+## 변경 목록
+
+### 백엔드
+
+**`QuestionController`**
+- `@RequestParam(required = false) String excludeQuestionUuid` →
+  `@RequestParam(required = false) List<String> excludeQuestionUuids`
+
+**`RecommendationService`**
+- `recommend()` 시그니처: `UUID excludeQuestionUuid` → `List<UUID> excludeQuestionUuids`
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)